src/query/graphite/native/functions.go

```diff
@@ -40,6 +40,36 @@ var (
 	functions = map[string]*Function{}
 )
 
+// list of graphite function name strings. (not whole list, update on-demand)
+const (
+	Average        = "average"
+	AverageSeries  = "averageSeries"
+	Avg            = "avg"
+	Count          = "count"
+	CountSeries    = "countSeries"
+	Current        = "current"
+	Diff           = "diff"
+	DiffSeries     = "diffSeries"
+	Empty          = ""
+	Last           = "last"
+	LastSeries     = "lastSeries"
+	Max            = "max"
+	MaxSeries      = "maxSeries"
+	Min            = "min"
+	MinSeries      = "minSeries"
+	Multiply       = "multiply"
+	MultiplySeries = "multiplySeries"
+	Range          = "range"
+	RangeOf        = "rangeOf"
+	RangeOfSeries  = "rangeOfSeries"
+	Stdev          = "stdev"
+	Stddev         = "stddev"
+	StddevSeries   = "stddevSeries"
+	Sum            = "sum"
+	SumSeries      = "sumSeries"
+	Total          = "total"
```

Comment:
nit: don't think these are exported anywhere, can probably keep em package local here 👍

Reply:
I changed them all to lowercase.

But then I had some duplicate declarations, so also add `FnName` to the end of each one. 

Hope this is OK.